internal/controller/scheduler/scheduler.go

```diff
@@ -91,9 +91,11 @@ func (scheduler *Scheduler) Run() {
 func (scheduler *Scheduler) reportStats(workers []v1.Worker, vms []v1.VM) {
 	for _, worker := range workers {
 		if worker.Offline(scheduler.workerOfflineTimeout) {
-			workersStat.With(map[string]string{"status": "offline"}).Inc()
+			workersStat.With(map[string]string{"worker_name": worker.Name, "status": "online"}).Set(0)
```

Comment:
Is there a need to have `Set(0)` metrics here and below?

I thought we could get away with just `Set(1)` and an appropriate labels.

Reply:
I agree, but it was feedback further up this PR to add this.